src/app/pages/page-title/page-title.ts

```diff
@@ -8,5 +9,15 @@ export class ComponentPageTitle {
   _title = '';
 
   get title(): string { return this._title; }
-  set title(title: string) { this._title = title; }
+
+  set title(title: string) {
+    this._title = title;
+    if (title !== '') {
+      title = ` - ${title}`;
+    }
+    this.bodyTitle.setTitle(`Angular Material${title}`);
```

Comment:
The order should be the other way around, e.g.
`"Autocomplete | Angular Material"`

This is better because people can have multiple tabs open and can easily see which tab is which without switching to it. Also use `|` instead of `-`
(see this pattern in-action at, e.g., https://developers.google.com/web/fundamentals/performance/prpl-pattern/)